jupyterlab_rtc_hub_settings/LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```

Comment:
Typically we run MIT on everything, at least I do. Is there a specific reason we need to use BSD?

Reply:
Nah, just a holdover from the cookiecutter template https://github.com/jupyterlab/extension-cookiecutter-ts/blob/3.0/%7B%7Bcookiecutter.python_name%7D%7D/LICENSE